.github/workflows/CI.yml

```diff
@@ -405,6 +405,18 @@ jobs:
       - name: Build CI
         run: yarn build-ci
 
+      - name: Run adapters tests
+      - run: yarn workspace @hicommonwealth/adapters test
+
+      - name: Run chains tests
+      - run: yarn workspace @hicommonwealth/chains test
```

Comment:
don't think this is going to work, chains don't have any tests yet

Reply:
Good point! They had the package.json script for it but no corresponding tests. I have removed it.